wolfcrypt/src/evp.c

```diff
@@ -7085,9 +7587,18 @@ int wolfSSL_EVP_MD_type(const WOLFSSL_EVP_MD* type)
         WOLFSSL_ENTER("wolfSSL_EVP_Cipher");
 
         if (ctx == NULL || ((src == NULL || dst == NULL) &&
-            (ctx->cipherType != AES_128_GCM_TYPE &&
+            (TRUE 
```

Comment:
```8 bit, trailing spaces or tabs:
./wolfcrypt/src/evp.c:7590:            (TRUE
```

Reply:
The TRUE looks like it is there to make the comparison still work because several clauses have guards around them.